src/runtime.js

```diff
@@ -87,6 +88,7 @@ export function installRuntimeServices(global) {
   installViewerService(global);
   installViewportService(global);
   installHistoryService(global);
+  installStorageService(global);
```

Comment:
Ok, let's leave here, but add `TODO` and link the factory-instances GitHub issue that Malte has created. Let's make sure we do it very soon.